crates/node-file-trace/src/lib.rs

```diff
@@ -437,6 +398,8 @@ pub async fn start(
                 println!("graph.html written");
             }
         },
+        module_options,
```

Comment:
Looks like the `cfg(feature = "persistent_cache")` guarded `run` above this isn't updated?

Reply:
`cfg(feature = "persistent_cache")` hasn't been maintained for a long time, and couldn't be compiled already. I think it's okay to leave it here and refactor it when we have a plan for it.